react/features/base/conference/reducer.js

```diff
@@ -206,7 +203,7 @@ function _conferenceJoined(state, { conference }) {
  * reduction of the specified action.
  */
 function _conferenceLeftOrWillLeave(state, { conference, type }) {
-    let nextState = state;
+    const nextState = { ...state };
```

Comment:
@saghul, this way you'll always cause the redux state to change even if the action doesn't get reduced to an actual change. The previous approach would create multiple copies until it settled on the final next redux state so it wasn't perfect by any means. At least it wouldn't cause a redux state change unless necessary. But the actions are more likely to cause a change than not so I guess we can go this way.

Reply:
I decided to go with this exactly because of the multiple copies. I think this specific case warrants it.